src/PKSim.Presentation/Presenters/Main/StatusBarPresenter.cs

```diff
@@ -237,9 +241,9 @@ private void updateJournalInfo(string name, string path, bool enabled)
 
       public void Handle(ProgressDoneWithMessageEvent eventToHandle)
       {
-         var message = $"{activeSimulationsCount} {eventToHandle.Message}";
+         var message = $"{_interactiveSimulationRunner.ActiveSimulationsCount} {eventToHandle.Message}";
```

Comment:
that;s not correct. I will still say 1 if there is only one and we had said we did not want to show it like this right? Did it make it like so in 12.1?